### DISCUSSION TITLE
Message Content to become Privileged

### DISCUSSION THREAD

**Discussion by Doomsdayrs** [kordlib/kord#472] [Meta]
https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Access-Deprecation-for-Verified-Bots

Placing this here for discussion, as this will effect large bots

**Comment by DRSchlaubi** [comment]
We'll do our best to have all of the upcoming Slash Command changes implemented before this happens, but since this isn't an issue, shouldn't we make this an actual Discussion?

**Comment by HopeBaron** [comment]
I moved this to discussion, we can't do anything right now as it seems that this intent is explicitly given by Discord.

**Comment by DRSchlaubi** [comment]
I don't think there is a lot to discuss here anymore

**Comment by lukellmann** [comment]
Message Content now is a Privileged Intent since more than [six months](https://discord.com/developers/docs/change-log#message-content-is-a-privileged-intent) and we haven't heard of any Kord-related issues with that since then. I think it's time to close this discussion.
